code/game/gamemodes/cult/ritual.dm

```diff
@@ -210,20 +209,19 @@
 			if(!(CULT_ELDERGOD in cult_mode.objectives) || !(CULT_SLAUGHTER in cult_mode.objectives))
 				to_chat(user, "<span class='warning'>[cult_mode.cultdat.entity_name]'s power does not wish to be unleashed!</span>")
 				return 0
-			var/confirm_final = alert(user, "This is the FINAL step to summon your dietys power, it is a long, painful ritual and the crew will be alerted to your presence", "Are you prepared for the final battle?", "My life for Nar-Sie!", "No")
-			if(confirm_final == "No")
-				to_chat(user, "<span class='cult'>You decide to prepare further before scribing the rune.</span>")
-				return
-			else
-				return 1
+		var/confirm_final = alert(user, "This is the FINAL step to summon your dietys power, it is a long, painful ritual and the crew will be alerted to your presence", "Are you prepared for the final battle?", "My life for [cult_mode.cultdat.entity_name]!", "No")
+		if(confirm_final == "No")
+			to_chat(user, "<span class='cult'>You decide to prepare further before scribing the rune.</span>")
+			return 0
+		else
+			return 1
 	else//the game mode is not cult..but we ARE a cultist...ALL ON THE ADMINBUS
-		if(!canbypass == 1)//not an admin-tome, check things
-			var/confirm_final = alert(user, "This is the FINAL step to summon your dietys power, it is a long, painful ritual and the crew will be alerted to your presence", "Are you prepared for the final battle?", "My life for Nar-Sie!", "No")
-			if(confirm_final == "No")
-				to_chat(user, "<span class='cult'>You decide to prepare further before scribing the rune.</span>")
-				return 0
-			else
-				return 1
+		var/confirm_final = alert(user, "This is the FINAL step to summon your dietys power, it is a long, painful ritual and the crew will be alerted to your presence", "Are you prepared for the final battle?", "My life for [cult_mode.cultdat.entity_name]!", "No")

```

Comment:
The second copy of the confirmation that starts on this line is unnecessary. Move the confirmation check above out of the `if(GAMEMODE_IS_CULT)` block and it will work the same for both cases.